velox/expression/VectorUdfTypeSystem.h

```diff
@@ -666,6 +710,10 @@ struct VectorReader<Row<T...>> {
     return false;
   }
 
+  inline bool mayHaveNullsRecursive() const {
```

Comment:
setChildrenMayHaveNulls is not defined for this reader. 
its ok that we dont cache anything but if the children are map or arrays this might be issue 
since we would throw.

Reply:
You're right, I should be calling it recursively on child readers